### ISSUE TITLE
ISIC Taxonomy

### ISSUE THREAD

**Issue by Rafiot** [MISP/misp-taxonomies#77]
Not sure if relevant for MISP, but it would be relatively easy to represent: https://unstats.un.org/unsd/cr/registry/regcst.asp?Cl=27

Note that it goes 4 levels depth: https://unstats.un.org/unsd/cr/registry/regcs.asp?Cl=27&Lg=1&Co=0111

**Comment by adulau** [comment]
Indeed, it's relevant. Interesting maybe we should add this one as a galaxy cluster like un-sector and it's very good compared to the [sector](https://github.com/MISP/misp-galaxy/blob/master/clusters/sector.json) which contain no description.  

@Delta-Sierra could you have a look? maybe create a new one or ask CERT-EU if the sector one can be merged together.
